packages/asl-pages/pages/project/replace-hba/index.js

```diff
@@ -0,0 +1,66 @@
+const { page } = require('@asl/service/ui');
+const { form } = require('../../common/routers');
+const schema = require('../../task/schema/upload-hba');
+const FormData = require('form-data');
+const { default: axios } = require('axios');
+const { UnauthorisedError } = require('@asl/service/errors');
+
+module.exports = (settings) => {
+  const app = page({
+    ...settings,
+    root: __dirname
+  });
+
+  app.use((req, res, next) => {
+    if (!req.user.profile.asruUser && !req.user.profile.asruAdmin) {
+      return next(
+        new UnauthorisedError(
+          'Only ASRU users can upload harm benefit analysis file'
+        )
+      );
+    }
+    next();
+  });
```

Comment:
```suggestion
  app.use(async (req, res, next) => {
    if (await req.user.can('project.replaceHBA');) {
      return next(
        new UnauthorisedError(
          'Only ASRU users can upload harm benefit analysis file'
        )
      );
    }
    next();
  });
```

I think this can be omitted as https://github.com/UKHomeOffice/aspel-workspace/pull/204/files#diff-9358b4d9939f3e879e5101849e6ff851e1c4781e9a8606ff235ed19145b04601R71 is already causing this check to happen.